packages/expo/src/Expo.ts

```diff
@@ -37,6 +37,7 @@ export { Print } from 'expo-print';
 export { Accelerometer, Gyroscope, Magnetometer, MagnetometerUncalibrated } from 'expo-sensors';
 import * as SMS from 'expo-sms';
 export { SMS };
+export { Speech } from 'expo-speech';
```

Comment:
I think we would like to keep the `import *, export` notation to let dead-code eliminators do their job